### DISCUSSION TITLE
Dva a více balíků

### DISCUSSION THREAD

**Discussion by mpavelcz** [Salamek/PplMyApi#73] [Q&A] [answered]
Ahoj,
snažím se vytvořit štítek na více balíků

$pocet_stitku = 2;
$package = new Package($packageNumber,  Product::PPL_PARCEL_CZ_PRIVATE, 'Testovaci balik', $recipient, $cityRouting, null, null, null, null, [], [], [], null, null, (int)$pocet_stitku );

chyba:
Salamek\PplMyApi\Model\Package::__construct(): Argument #15 ($packageSet) must be of type ?Salamek\PplMyApi\Model\IPackageSet, int given

můžete mě prosím trochu nakopnout, kde dělám chybu?

Díky

**Comment by Salamek** [comment, marked as answer]
Je to napsane primo v te error message, 15 argument `Salamek\PplMyApi\Model\Package::__construct()`  je typu `Salamek\PplMyApi\Model\IPackageSet` a vy tam davate `int`...

Zhruba nejak takhle:

```php
use Salamek\PplMyApi\Model\PackageSet;
use Salamek\PplMyApi\Model\Package;

$mainPackageNumber = 'MAIN_PACKAGE_NUMBER';
$packageSet = new PackageSet($mainPackageNumber, 1, 2); // For constructor parameters see https://github.com/Salamek/PplMyApi/blob/master/src/Model/PackageSet.php
$package = new Package($mainPackageNumber, Product::PPL_PARCEL_CZ_PRIVATE, 'Testovaci balik', $recipient, $cityRouting, null, null, null, null, [], [], [], null, null, $packageSet);

$packageSet->setPackagePosition(2);
$package2 = new Package($packageNumber2, Product::PPL_PARCEL_CZ_PRIVATE, 'Testovaci balik2', $recipient, $cityRouting, null, null, null, null, [], [], [], null, null, $packageSet);


try
{
    $pplMyApi->createPackages([$package, $package2]);
}
catch (\Exception $e)
{
    echo $e->getMessage() . PHP_EOL;
}

```

**Reply by mpavelcz** [reply]
do této fáze PackageSet jsem se už také dostal, jen jsem nepřidál druhý $package2, moc díky za nakopnutí
